packages/components/src/templates/next/components/internal/Filter/Filter.tsx

```diff
@@ -45,7 +45,7 @@ const Filter = ({
               <label
                 key={itemId}
                 htmlFor={itemId}
-                className="flex w-full flex-row rounded px-1 py-2 align-middle hover:bg-interaction-main-subtle-hover has-[:focus]:ring-2 has-[:focus]:ring-focus-outline"
+                className="flex w-full flex-row rounded px-1 py-2 align-middle has-[:focus]:ring-2 has-[:focus]:ring-focus-outline hover:bg-interaction-main-subtle-hover"
```

Comment:
nit: the alignment for the filter checkbox isn't ideal, not middle aligned

<img width="209" alt="Screenshot 2024-08-16 at 10 44 33" src="https://github.com/user-attachments/assets/db4e5a2c-62bd-4acf-baef-cd1570251171">

Reply:
this one is old filter you write one HAHA. Just added format. The new filter in next PR